### PR TITLE
POM-89: Migrate 'created_at' dates

### PR DESCRIPTION
**Pull request by TobyRet**
- Created Rake task that migrates Allocation 'created_at' dates to the 'primary_pom_allocated_at' date in the corresponding AllocationVersion record

### CODE REVIEW

File: spec/factories/allocation.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+require 'faker'
```
Comment (rossjones) [inline spec/factories/allocation.rb]: Is this meant to be in this PR?

File: spec/services/date_migrater_spec.rb [hunk 2]
```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+require 'byebug'
```
Comment (rossjones) [inline spec/services/date_migrater_spec.rb]: byebug?

File: spec/factories/allocation.rb (hunk 1 above)
Comment (TobyRet) [reply spec/factories/allocation.rb]: The Faker gem is for creating dummy data. Once the migration is run this factory will be deleted

Comment (TobyRet) [reply spec/factories/allocation.rb]: Yep (it's for creating dummy data, however this factory will be removed once the migration is run)

**Pull request merged**